nin/frontend/app/scripts/filters/errorExcerpt.js

```diff
@@ -1,5 +1,9 @@
 function errorExcerpt($sce) {
   return function(error, fileCache) {
+    if (fileCache === undefined || !error.path in fileCache) {
+      return;
```

Comment:
Does `error` just disappear when we do this? Shoud we perhaps throw the error or something to compensate?